src/uwtools/drivers/orog.py

```diff
@@ -37,7 +37,7 @@ def grid_file(self):
         The input grid file.
         """
         grid_file = Path(self.config["grid_file"])
-        yield self.taskname("Input grid file")
+        yield self.taskname(f"Input grid file {grid_file}")
```

Comment:
I was finding it hard to know which file I was missing during the debugging process, so added the path here.